### ISSUE TITLE
--column option

### ISSUE THREAD

**Issue by b4b4r07** [monochromegane/the_platinum_searcher#92]
I'd like you to add `--column` option like that `ag` has.

```console
$ ag --nogroup --column split
autoload/ag.vim:22:7:  "if split(system("ag --version"), "[ \n\r\...
```

```console
$ pt --nogroup split
autoload/ag.vim:22:  "if split(system("ag --version"), "[ \n\r\...
```

**Comment by syohex** [comment]
`--column` option is already implemented at #79.
Do you use latest version ?

**Comment by b4b4r07** [comment]
```console
[b4b4r07]% go get github.com/monochromegane/the_platinum_searcher
[b4b4r07]% pt --version
1.7.6
[b4b4r07]% pt --column
unknown flag `column'
[b4b4r07]%
```
